src/test/ui/methods/method-ambig-one-trait-unknown-int-type.stderr

```diff
@@ -23,6 +23,10 @@ LL | impl Foo for Vec<usize> {
 ...
 LL | impl Foo for Vec<isize> {
    | ^^^^^^^^^^^^^^^^^^^^^^^
+help: try using a fully qualified path to specify the expected types
+   |
+LL |     <Vec<T> as Foo>::foo(&x);
+   |     ++++++++++++++++++++++ ~
```

Comment:
This should have been `<Vec<isize> as Foo>::foo(&x)`, but that'd be outside of scope for this PR.